### ISSUE TITLE
Not Backing Up Messages

### ISSUE THREAD

**Issue by bennyprax** [Androz2091/discord-backup#169]
I was trying this out today. First off, the backup folder is directly inside the lib folder of the `discord-backup`. 

Secondly, when I checked the messages for each chat it had empty `[]` lists on them. 

Not sure if this is the problem of discord-backup or the backups bot. Let me know if I should post this issue there instead of here. Thanks!

**Comment by Androz2091** [comment]
> First off, the backup folder is directly inside the lib folder of the discord-backup

Yes, it is the excepted behavior until you change it using `setBackupFolder()` 👍 

> Secondly, when I checked the messages for each chat it had empty [] lists on them.

I can't help you without your code, can you send it please?

> Not sure if this is the problem of discord-backup or the backups bot. Let me know if I should post this issue there instead of here. 

Are you using the backups-bot code (Androz2091/backups-bot repository), or are you trying to make your own bot using the library?

**Comment by bennyprax** [comment]
Hi!  So I'm using the backup-bots repository.

**Comment by aditya-learns** [comment]
Please share your code

**Comment by Detergent13** [comment]
> Yes, it is the excepted behavior until you change it using `setBackupFolder()` 👍

Just wanted to note that it's (now?) **setStorageFolder() instead.** Took me a sec to realise so figured I'd hopefully save others some time!

**Comment by Sorok-Dva** [comment]
Hello, having the same problem here, the channels are rightly restored but the messages doesn't come with, even if they're in the json file
